BackEndApp/BackEndApp/settings.py

```diff
@@ -125,3 +127,34 @@
 EMAIL_HOST_USER = os.environ.get("EMAIL_HOST_USER")
 DEFAULT_FROM_EMAIL = os.environ.get("EMAIL_HOST_USER")
 EMAIL_HOST_PASSWORD = os.environ.get("EMAIL_HOST_PASSWORD")
+
+logging.config.dictConfig({
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'console': {
+            'format': '%(name)-12s %(levelname)-8s %(message)s'
```

Comment:
For the logging formats, it would be great if you could use the one mentioned in this issue #199. It's just that the formatting looks a bit neat and consistent.